README.md

```diff
@@ -1,13 +1,163 @@
-# tools
+# Tools for authoring and serving codelabs
+Alex Vaghin, [@x1ddos](https://twitter.com/user/x1ddos)
+Marc Cohen, [mcohen.io](https://mcohen.io), [@marcacohen](https://twitter.com/user/marcacohen)
 
-Codelabs management & hosting tools.
-See [claat](claat) directory for detailed description of the `claat` command
-line tool.
+![](https://raw.githubusercontent.com/googlecodelabs/tools/master/claat/demo.gif)
 
-Codelabs can be authored in Google Docs, using some simple formatting conventions. 
-In the near future, we're also planning to add codelab authoring via markdown syntax.
-This repo provides the tools necessary to author, generate, and serve your own codelabs.
+Codelabs are interactive instructional tutorials, which can be authored in Google Docs
+using some simple formatting conventions. You can also author codelabs using markdown syntax.
+This repo contains all the tools and documentation you’ll need for building and publishing
+your own codelabs.
 
 If you're interested in authoring codelabs, join [this group](https://groups.google.com/forum/#!forum/codelab-authors),
-which connects you with other authors and provides access to the 
-[Codelab Formatting Guide](https://docs.google.com/document/d/18dnMdUJQaGKY1Tit_-fO1YOpOpAbA4hh0YDXQlCEjvA/edit?ts=574ec5d9).
+which connects you with other authors and provides access to the [Codelab Formatting Guide](https://docs.google.com/document/d/18dnMdUJQaGKY1Tit_-fO1YOpOpAbA4hh0YDXQlCEjvA/edit?ts=574ec5d9).
+
```

Comment:
Replace the doc URL with https://g.co/codelabs/guide shortcut? I saw it down below.

Reply:
fixed.